sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -138,8 +139,7 @@ def foo((a, b)):
 
 _ANY_VAR_POSITIONAL = typehints.Tuple[typehints.Any, ...]
 _ANY_VAR_KEYWORD = typehints.Dict[typehints.Any, typehints.Any]
-# TODO(BEAM-8280): Remove this when from_callable is ready to be enabled.
-_enable_from_callable = False
+_disable_from_callable = False
```

Comment:
Should this be True? (based on the check in line 234)

Reply:
No, I inverted the logic and made using type annotations on by default.